lib/active_model/serializer/adapter.rb

```diff
@@ -1,7 +1,10 @@
+require 'active_model/serializer/cache'
+
 module ActiveModel
   class Serializer
     class Adapter
       extend ActiveSupport::Autoload
+      include ActiveModel::Serializer::Cache
```

Comment:
The loss would be result of passing the block correspondent to the yield to order function. But yeah it's nothing big.

---------------------------------

Agreed. Even ```cached_object`` isn't a good name, I'm working on this too.